src/ttsim/plot/dag/tt.py

```diff
@@ -65,14 +66,16 @@ def tt(
     primary_nodes
         The qnames or paths of the primary nodes. Primary nodes are used to determine
         which other nodes to include in the plot based on the selection_type. They may
-        be root nodes (for descendants), end nodes (for ancestors), or middle nodes
-        (for neighbors). If not provided, the entire DAG is plotted.
+        be root nodes (for descendants), end nodes (for ancestors), or middle nodes (for
+        neighbors). If not provided, the entire DAG is plotted.
     selection_type
         The type of the DAG to plot. Can be one of:
             - "neighbors": Plot the neighbors of the primary nodes.
             - "descendants": Plot the descendants of the primary nodes.
             - "ancestors": Plot the ancestors of the primary nodes.
-            - "nodes": Plot the primary nodes only.
+            - "all_paths": Plot all paths that connect primary nodes. Primary nodes are
```

Comment:
I guess this is simply the behaviour of networkx? If easy, I think I'd prefer having all primary nodes in the plot, even if they are not connected to any of the others.

A specific test for this (either way that we end up choosing) would be good.

Reply:
I'd prefer the orphaned nodes -- too easy to think you have done something wrong if nothing is shown. Agreed on the impurity in the naming, but that's the lesser evil IMO.